packages/cli/src/commonlib/appStudioLoginUserPassword.ts

```diff
@@ -90,22 +90,6 @@ export class AppStudioTokenProviderUserPassword implements AppStudioTokenProvide
         });
     }
 
-    async setStatusChangeCallback(
-      statusChange: (status: string, token?: string, accountInfo?: Record<string, unknown>) => Promise<void>
-    ): Promise<boolean> {
-        return new Promise((resolve) => {
-            resolve(true);
-        });
-    }
-
-    public async getStatus(): Promise<LoginStatus> {
```

Comment:
CLI wants to use `getStatus()`. Maybe you delete it by mistake?

Reply:
Already add it back, thanks.